data-es7.js

```diff
@@ -78,6 +78,16 @@ exports.browsers = {
   firefox41: {
     full: 'Firefox',
     short: 'FF 41',
+    obsolete: true,
```

Comment:
Instead of that, we could just store browsers in a separate requireable file, no?